test/validation/api/event/event_vector.c

```diff
@@ -52,6 +52,25 @@ static odp_pool_t create_buffer_pool(uint32_t num_items)
 	return pool;
 }
 
+static odp_event_vector_t event_vector_alloc(void)
+{
+	odp_event_vector_t evv;
+	odp_event_t *ev_tbl = NULL;
+	uint32_t size;
+
+	evv = odp_event_vector_alloc(evv_pool);
+	CU_ASSERT_FATAL(evv != ODP_EVENT_VECTOR_INVALID);
+
+	CU_ASSERT(odp_event_vector_size(evv) == 0);
+	CU_ASSERT(odp_event_vector_type(evv) == ODP_EVENT_ANY);
+	CU_ASSERT(odp_event_vector_user_flag(evv) == 0);
+	size = odp_event_vector_tbl(evv, &ev_tbl);
+	CU_ASSERT(size == 0);
```

Comment:
Could also check event vector source pool. `odp_event_vector_pool()` is currently untested.

Reply:
Added an assert for that after alloc in v5